cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,34 @@
+//// <reference types='cypress' />
+//// <reference types='../support' />
```

Comment:
There are duplicate `/// <reference types='cypress' />` declarations. The one on line 4 is redundant and should be removed to clean up the code.